incompatibility/incomp-scanning_test.go

```diff
@@ -87,6 +87,32 @@ func TestIncompatibilityExistence(t *testing.T) {
 	}
 }
 
+// Test verifying detailing proces of an incompatibility report
+func TestDetailing(t *testing.T) {
```

Comment:
Can you test this from the public API?

Reply:
refactored incomp-scanning and added tests to examine report writing process to have more test coverage on gnostic flow

Reply:
I see we are still testing against a private method detailReport.  Can we test against the public method GnosticIncompatibiltyScanning?  

That being said GnosticIncompatibiltyScanning is doing minimal work and is more of an adaptor, so testing detailReport directly is reasonable. 

Thoughts?

Reply:
Best practices that tests are named TestMethodName